### PR TITLE
BAVL-439 removal of all mirgation specfic code now that the migration of old bookings to new has been done in production.

### PR DESCRIPTION
**Pull request by matt-crowson-moj**
Goodbye migration code.

Now that we have migrated the old BVLS service data into the re-platformed service we no longer need the migration specific code.

### CODE REVIEW

File: migration/display-maintenance-screen.sh [hunk 1]
```diff
@@ -1,41 +0,0 @@
...
-type: Opaque
-stringData:
-  maintenance_mode: "true"
-  maintenance_start_datetime: "$STARTTIME"
-  maintenance_end_datetime: "$ENDTIME"
-EOF
-
-# Apply the secret
-kubectl apply -f ./$SECRETS_FILE
-echo "Applied secrets to $NAMESPACE"
```
Comment (timharrison-moj) [inline migration/display-maintenance-screen.sh:26]: Matt - can we keep this one please? Think we'll want the ability to put up the screen..

File: migration/feature-switch-statuses.sh [hunk 2]
```diff
@@ -1,58 +0,0 @@
...
-  echo "BOOK_A_VIDEO_LINK_FEATURE_TOGGLE_ENABLED=$BOOK_A_VIDEO_LINK_FEATURE_TOGGLE_ENABLED"
-}
-
-function bookAVideoLink() {
-  MAINTENANCE_MODE=$(kubectl -n "hmpps-book-a-video-link-$ENVIRONMENT" get secret maintenance -o jsonpath='{.data.maintenance_mode}' | base64 -d)
-
-  echo "hmpps-book-a-video-link in environment $ENVIRONMENT"
-  echo
-  echo "maintenance_mode=$MAINTENANCE_MODE"
-}
```
Comment (timharrison-moj) [inline migration/feature-switch-statuses.sh:45]: And this part of the script please?

File: migration/remove-maintenance-screen.sh [hunk 3]
```diff
@@ -1,33 +0,0 @@
...
-# Create the secrets YAML file
-cat <<EOF > $SECRETS_FILE
-apiVersion: v1
-kind: Secret
-metadata:
-  name: maintenance
-  namespace: $NAMESPACE
-type: Opaque
-stringData:
-  maintenance_mode: "false"
```
Comment (timharrison-moj) [inline migration/remove-maintenance-screen.sh:17]: And this..

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/nomismapping/NomisMappingClient.kt [hunk 4]
```diff
@@ -13,16 +13,6 @@ class NomisMappingClient(private val nomisMappingApiWebClient: WebClient) {
...
 
-  @Deprecated(message = "Can be removed when migration is completed")
-  fun getNomisLocationMappingBy(internalLocationId: Long): NomisDpsLocationMapping? = nomisMappingApiWebClient
-    .get()
-    .uri("/api/locations/nomis/{id}", internalLocationId)
-    .retrieve()
-    .bodyToMono(NomisDpsLocationMapping::class.java)
-    .doOnError { error -> log.info("Error looking up internal location mapping by nomis location id $internalLocationId in mapping service", error) }
-    .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
-    .block()
```
Comment (timharrison-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/nomismapping/NomisMappingClient.kt:24]: Don't we still need to map location codes?

**Review by timharrison-moj — COMMENTED** [review]
Can we keep the maintenance screen ability please? And a question about the location mapping - still used?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/nomismapping/NomisMappingClient.kt (hunk 4 above)
Comment (matt-crowson-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/nomismapping/NomisMappingClient.kt:24]: No this was for migration only. Old booking has nomis location ID, this tells us the mapping for the new UUID for migrated bookings.

File: utils/.DS_Store
Comment (timharrison-moj) [inline utils/.DS_Store:1]: Needed?

**Pull request closed without merging**